gh-133895: provide C99 Annex G return values for cmath's functions (#134995)
python/cpython: Doc/library/cmath.rst

      1.4142135623730951j

> Most functions computes and returns the C99 Annex G recommended result.  If the

Reviewer: ```suggestion
Most functions compute and return the C99 Annex G recommended result.  If the
```

standard recommends raising ``FE_DIVBYZERO`` or ``FE_INVALID`` floating-point

> exceptions --- the :exc:`ValueError` is raised and recommended result available
> as the ``value`` attribute of the exception object.

Reviewer: ```suggestion
exceptions --- a :exc:`ValueError` is raised and the recommended result
is available as the ``value`` attribute of the exception object.
```

      1.4142135623730951j

Most functions compute and return the C99 Annex G recommended result.  If the
standard recommends raising ``FE_DIVBYZERO`` or ``FE_INVALID`` floating-point
exceptions --- a :exc:`ValueError` is raised and the recommended result is
available as the ``value`` attribute of the exception object.  If range error
occurs due to overflow in any component of the result --- a :exc:`OverflowError`

> is raised.

Reviewer: ```suggestion
Most functions compute and return the C99 Annex G recommended result.
If the standard recommends raising ``FE_DIVBYZERO`` or ``FE_INVALID``
floating-point exceptions, a :exc:`ValueError` is raised and the recommended
result is available as the ``value`` attribute of the exception object.
If a range error occurs due to an overflow in any component of the result,
an :exc:`OverflowError` is raised.
```

Sorry, I misread the paragraph in the first round of review.
